File: demo-flutter-miniapp/vietsuper_android/app/src/main/java/com/vietsuper/host/MiniApp.kt
package com.vietsuper.host

data class MiniApp(
    val id: String,
    val name: String,
    val emoji: String,
    /** File Dart chứa hàm main của mini này — cùng địa chỉ với bản iOS. */
    val library: String,
)

val miniApps = listOf(
    MiniApp("food", "Đồ ăn", "🍜", "package:mini_flutter/main_food.dart"),
)

File: demo-flutter-miniapp/vietsuper_ios/VietSuper/MiniApp.swift
import Foundation

struct MiniApp: Identifiable {
  let id: String
  let name: String
  let emoji: String
  /// File Dart chứa hàm main của mini này.
  let library: String
}

let miniApps: [MiniApp] = [
  MiniApp(id: "food", name: "Đồ ăn", emoji: "🍜",
          library: "package:mini_flutter/main_food.dart"),
]
